Sub/Src/ComputerVision/Nodes/CamNode.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
+import cv2
+import math
+import random
+import socket
+import sys
+import struct
+import math
+import io
+import time
+from ctypes import *
+from libs.darknet import *
+from MechOS.message_passing.Nodes.node_base import node_base
+
+class CamNode(node_base):
+
+    def __init__(self, MEM, IP):
+
+        # IP and MEM RAM locations
+        node_base.__init__(self, MEM, IP)
+
+        # Instantiations
+
+        #--MESSAGING INFO--#
+        self.MAX_UDP_PACKET_SIZE = 1500
+
+        # The end byte of the image sent over udp
+        self.END_BYTE = bytes.fromhex('c0c0')*2
+
+        # the Max Packet Size the Port will accept
+        self.MAX_PACKET_SIZE = 1500
+
+        #--CAMERA INSTANCE--#
+        self.capture = cv2.VideoCapture(1)
+
+        # Maximum image size the Tegra allows without timeout
+        self.capture.set(3, 450)
+        self.capture.set(4, 450)
+
+        # Neural Network Loaded from instance in darknet module
+        self.net  = load_net(b'./libs/darknet/cfg/yolov3-tiny.cfg',
+                       b'./libs/darknet/weights/yolov3-tiny.weights',
+                       0)
+
+        self.meta = load_meta(b'./libs/darknet/cfg/coco.data')
```

Comment:
Also make sure to specify the mechatronics data file and not the standard one.

Reply:
@senseishafi This is you sir. 

> Just a few suggestions, also I think it would be best to add darknet as a submodule instead of just committing the code

The darknet subdirectory, is composed of weights configs and data necessary for running Yolo. We can add less of these, and eliminate the option for trainging, because we don't need all of them, and won't be training on the tegra. but what do you mean by adding as a submodule?